app/services/edge_cache/bust_comment.rb

```diff
@@ -0,0 +1,32 @@
+module EdgeCache
+  class BustComment < Bust
```

Comment:
see my comment about merging `BustComment` and `BustCommentable` into `BustCommentable`: after all we're busting the object the comment is attached to and its comments, not "a comment"